site/docs/tutorials/snake/moving-the-snake.md

```diff
@@ -136,6 +136,55 @@ Now if you execute this, you'd notice that you can't see much. In fact, you migh
 
 </Page>
 
+<Page value="wat">
+
+To achieve the first step (moving the body, excluding the head), a loop is all you need:
+
+```wasm
+  (func $snake-update
+    (local $offset i32)
+
+    ;; loop backward from the end to the beginning.
+    (local.set $offset
+      (i32.mul
+        (i32.sub (i32.load (i32.const 0x19a8)) (i32.const 1))
+        (i32.const 8)))
+
+    (loop $loop
+      ;; body[i].x = body[i - 1].x
+      ;;
+      ;; The - 8 offset is baked into the i32.load offset.
+      (i32.store offset=0x19a4
```

Comment:
Can you check this offset, too? I followed along the tutorial till this part and I had to adjust the offset here to `offset=0x19ac` to make the game match the tutorial.

Reply:
Yes, all these offsets were off by 8. Thanks!